src/core/ColorPicker/ColorPicker.test.tsx

```diff
@@ -0,0 +1,61 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react';
+import { ColorPicker } from './ColorPicker';
+import ColorSwatch from './ColorSwatch';
+
+it('should render in its most basic state', () => {
+  const { container } = render(
+    <ColorPicker>
+      <ColorSwatch key={1} color='#9BA5AF' />
+      <ColorSwatch key={2} color='#23450b' />
+      <ColorSwatch key={3} color='#00121D' />
+      <ColorSwatch key={4} color='#002A44' />
+    </ColorPicker>,
+  );
+
+  expect(container.querySelector(`.iui-color-picker`)).toBeTruthy();
+  expect(container.querySelector(`.iui-color-palette`)).toBeTruthy();
+  expect(container.querySelectorAll(`.iui-color-swatch`).length).toBe(4);
+});
+
+it('should render color picker with active color swatch', () => {
+  const { container } = render(
+    <ColorPicker>
+      <ColorSwatch key={1} color='#9BA5AF' isActive={true} />
+      <ColorSwatch key={2} color='#23450b' />
+    </ColorPicker>,
+  );
+
+  expect(
+    container.querySelectorAll(`.iui-color-swatch.iui-active`).length,
+  ).toBe(1);
+
+  const colorSwatches = container.querySelectorAll('.iui-color-swatch');
+  expect(colorSwatches[0].classList).toContain('iui-active');
```

Comment:
Added

Reply:
you don't really need this one because you're testing for `iui-active` on the swatch itself
![image](https://user-images.githubusercontent.com/9084735/134540657-4a1e025e-0918-4044-b593-3934aef3163d.png)

That's why I removed it in the suggested change above